UPGRADE-4.3.md

```diff
@@ -23,6 +23,7 @@ Config
 Form
 ----
 
+ * Support for using integers as form names is deprecated and will lead to a type error in 5.0, use strings instead.
```

Comment:
will it? automatic casting rules would turn the int to a string and all will be silent, no?